scripts/lua2dox.lua

```diff
@@ -24,11 +24,18 @@ Lua-to-Doxygen converter
 Partially from lua2dox
 http://search.cpan.org/~alec/Doxygen-Lua-0.02/lib/Doxygen/Lua.pm
 
-Running
+RUNNING
 -------
 
 This script "lua2dox.lua" gets called by "gen_vimdoc.py".
 
+DEBUGGING/DEVELOPING
+---------------------
+
+1. To debug, run gen_vimdoc.py with --keep-tmpfiles:
+   python3 scripts/gen_vimdoc.py -t treesitter --keep-tmpfiles
+2. The filtered result will be written to ./tmp-lua2dox-doc/….lua.c
```

Comment:
Now `gen_vimdoc.py ... --keep-tmpfiles` will write the lua2dox filtered quasi-C output for inspection.

(How were people debugging this before?)

Reply:
Trial and error, of course ;)